### ISSUE TITLE
all functionality back on current master?

### ISSUE THREAD

**Issue by thejonroberts** [DillGromble/wicTN#17]
I think we got this working again, but wanted to make an official issue in case that is not the case!

**Comment by itzsaga** [comment]
Yeah, I think you're talking about what I fixed with #12. If not let's figure it out and fix it!
